test/HelloWord.test.js

```diff
@@ -0,0 +1,9 @@
+const { assert } = require('chai');
+
+describe('Array', () => {
+  describe('#indexOf()', () => {
+    it('should return -1 when the value is not present', function() {
+      assert.equal(-1, [1,2,3].indexOf(4));
+    });
+  });
+});
```

Comment:
Agrega una linea al final del archivo = End Of File = EOF.

Reply:
El archivo ya tiene el EOF.